_layouts/hub-authenticated-emails.txt

```diff
@@ -1,2 +1 @@
-{% capture addrs %}{% for i in page.addrs %}{{ i }}{% unless forloop.last %}
-{% endunless %}{% endfor %}{% endcapture %}{{ addrs }}
+{{ page.addrs }}
```

Comment:
Because the loop was adding newlines before the move to 3.0.0.beta2, and now it's not adding any at all, putting all of the email addresses on one line with no spaces in between.